docs/installation.rst

```diff
@@ -45,54 +45,106 @@ Fedora 22
 .. _installing node.js on Red Hat: https://nodejs.org/en/download/package-manager/#enterprise-linux-and-fedora
 
 
+Conda
+^^^^^^^^^
+
+Conda installation and conda environment setup should be the first step in
+creating a development environment for Minerva.
+
+- Install Conda (https://conda.io/miniconda.html)
+
+- Create minerva development environment
+
+::
+
+    mkdir geoviz
+    git clone https://github.com/Kitware/minerva.git
+    cd minerva
+    conda env create -f conda_env.yml python=2.7
+    source activate minerva-dev
+
+- Run mongo daemon
+
+::
+
+    mongod &
+
+- Install Girder
+
+::
+
+    cd ..
+    git clone https://github.com/girder/girder.git
+    cd girder
+    pip install -e .
+    girder-install web
+
+Notes:
+
+- Make sure that conda minerva-dev environment is active (source activate minerva-dev)
+  for the next steps.
+
+
+Now proceed to setup Girder admin user and assetstore in the next section
+
 Setup Girder admin user and assetstore
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- `Run Girder`_ to ensure that it works.  Mongo should already be running, and you should follow the instructions for a source tree install.
+- `Run Girder`_ to ensure that it works.  Mongo should already be running,
+and you should follow the instructions for a source tree install (ignore
+if using conda environment instructions above).
 
 .. _Run Girder: http://girder.readthedocs.org/en/latest/installation.html#run
 - Navigate to Girder in your browser, register an admin user.
 - Navigate to the Admin console in Girder, when you are logged in as an admin user, then click on the Assetstores section.
 - Create a default Assetstore.
 
-Install database_assetstore as a Girder plugin
+Install KTile as a Girder plugin
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This is needed because minerva depends on database_assetstore plugin.
+This is needed because minerva depends on Girder KTile plugin.
 
-- Install girder_db_items plugin into the Girder plugins directory.
+- Install girder_ktile plugin.
 
 ::
 
-   cd GIRDER_DIR/plugins
-   git clone https://github.com/OpenGeoscience/girder_db_items database_assetstore
+    git clone https://github.com/OpenGeoscience/girder_ktile
+    girder-install plugin -s girder_ktile
+    girder-install web --dev --plugins girder_ktile
```

Comment:
Technically, when we build Minerva, it's dependencies (girder_ktile and database_assetstore) will be built too. So, do you think we need this step?

Reply:
that is a good point does it work all the time? I always had this issue where in some cases it did not. Did you experience the same?

Reply:
Yes, I think it always works for me. In fact, when I am testing this, I only run the Minerva one, and it worked good.